src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+body,
+ul {
+  margin: 0;
+}
+
+ul {
+  padding: 0;
+}
```

Comment:
🔥

Reply:
this approach was invented at one Q&A in order to force successful test passing and deployment of this task)